app/services/google_auth.rb

```diff
@@ -16,27 +16,31 @@ def service_name
       "google"
     end
 
+    def primary_key
+      :"#{service_name}_id_token"
+    end
+
     def expected_fields
       %w[sub email given_name family_name]
     end
 
     # no easy way to get decoded token without validation with the google library, so this will return it with validation,
     # that's ok in the execution flow, if it raises an exception it will be caught and returned in AuthServiceBase
-    def decode_token(access_token)
-      Google::Auth::IDTokens.verify_oidc access_token, aud: ENV.fetch("GOOGLE_APP_ID")
+    def decode_token(google_id_token)
+      Google::Auth::IDTokens.verify_oidc google_id_token, aud: ENV.fetch("GOOGLE_APP_ID")
     end
 
-    def validate_token!(access_token, _decode_token)
-      decode_token(access_token)
+    def validate_token!(google_id_token, _decoded_token)
+      decode_token(google_id_token)
     end
 
     def remote_user_id(decoded_token)
       decoded_token["sub"]
     end
 
-    def extract_user_atts(access_token, decoded_token)
+    def extract_user_atts(_google_id_token, decoded_token)
       {
-        google_user_id: remote_user_id(decoded_token),
+        google_id_token: remote_user_id(decoded_token),
```

Comment:
this should have stayed as `google_user_id`

Reply:
Same for this